scripts/check_licenses.py

```diff
@@ -0,0 +1,92 @@
+"""Check if license fields are valid in all records."""
```

Comment:
<!-- TODO's are small, trivial, but necessary changes. Distinguishing todo comments from issues: or suggestions: helps direct the reader's attention to comments requiring more involvement.  -->
**todo:** ‏Can you please introduce leading shebang?

```
#!/usr/bin/env python
```

And make this script executable by itself?

```console
chmod u+x scripts/check_licenses.py
```

This will be good for consistency with the other helper scripts.